tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AspNetCore/UserEvents/SignInManagerPasswordSignInUserIntegration.cs

```diff
@@ -63,12 +62,44 @@ public static class SignInManagerPasswordSignInUserIntegration
     internal static TReturn OnAsyncMethodEnd<TTarget, TReturn>(TTarget instance, TReturn returnValue, Exception exception, in CallTargetState state)
         where TReturn : ISignInResult
     {
-        if (!returnValue.Succeeded && Security.Instance is { TrackUserEvents: true } security)
+        if (Security.Instance is { TrackUserEvents: true } security)
```

Comment:
I know, I'm a grumpy old man, but my eyes get sore with this syntax 🙈

Reply:
its quite nice as it avoids the 2 steps of affecting Security.Instance to `security` variable then testing `TrackUserEvents`